src/script.js

```diff
@@ -0,0 +1,551 @@
+import menuList from "./models/menuModel.js";
```

Comment:
script.js ?? this file name is not at all descriptive, if not used you can delete from this branch.

Reply:
deleted